content/posts/scientific-python/translations/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Setting up and managing translations for Scientific Python projects"
+date: 2024-08-08
+draft: false
+tags:
+  [
+    "Scientific-Python",
+    "translations",
+    "inclusion",
+    "CZI",
+  ]
+description: "Setting up and managing translations for Scientific Python projects."
+displayInList: true
+author: ["Melissa Mendonça", "Albert Steppi", "Tania Allard", "Ralf Gommers", "Stéfan van der Walt", "Jarrod Millman"]
+---
+
+In November 2022, [the Chan Zuckerberg Initiative (CZI) awarded the Scientific Python project with a grant to improve communications infrastructure and accessibility](https://blog.scientific-python.org/scientific-python/2022-czi-grant/). This proposal involves several key areas to improve sustainability, inclusivity and accessibility of the Scientific Python ecosystem. One of these areas is making software documentation and user interfaces available in multiple languages. We are happy to announce that we are now able to develop and publish translations of multiple project websites.
```

Comment:
```suggestion
In November 2022, [the Chan Zuckerberg Initiative (CZI) awarded the Scientific Python project with a grant to improve communications infrastructure and accessibility](https://blog.scientific-python.org/scientific-python/2022-czi-grant/). This proposal involves several key areas to improve sustainability, inclusivity and accessibility of the Scientific Python ecosystem. One of these areas is making software documentation and user interfaces available in multiple languages. We are happy to announce that we  have developed the necessary infrastructure and process to allow translating multiple project websites.
```

The suggested wording should probably not used as-is. The gist of what I'm suggesting is making it clear that you have built the *capacity* to translate websites, not that you've translated several websites.

Reply:
Makes sense! Will rephrase.